okhttp-coroutines/src/main/kotlin/okhttp3/JvmCallExtensions.kt

```diff
@@ -42,7 +42,9 @@ suspend fun Call.executeAsync(): Response =
           call: Call,
           response: Response,
         ) {
-          continuation.resume(value = response, onCancellation = { call.cancel() })
+          continuation.resume(response) {
+            response.closeQuietly()
```

Comment:
I am naughty and did both a formatting change and a behavior change here.

Used to be cancel -- asynchronously interrupt a call without releasing any resources
Now it’s close -- synchronously release resources

Reply:
Should Retrofit borrow this improvement in `retrofit2/KotlinExtensions.kt`? @JakeWharton

Reply:
Retrofit doesn't have this problem. We never pass a live resource across the coroutine boundary. Our underlying request has already been completely consumed and closed before we resume.

Reply:
> Retrofit doesn't have this problem. We never pass a live resource across the coroutine boundary. Our underlying request has already been completely consumed and closed before we resume.

Got it, thanks